.github/workflows/release-publish.yml

```diff
@@ -20,26 +20,10 @@ jobs:
         env:
           NODE_OPTIONS: --max-old-space-size=8192
 
-  test_visual:
-    if: github.event.pull_request.merged == true
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v3
-      - uses: ./.github/actions/install
-      - run: pnpm run build-storybook
```

Comment:
I think we need to build a storybook here to publish screenshots in a Chromatic.
If not, need to check this.

Reply:
So we could just build it without running visual testing afterwards?